test/io/test_tim_grabber.cpp

```diff
@@ -0,0 +1,141 @@
+#include <pcl/test/gtest.h>
+#include <pcl/point_types.h>
+#include <pcl/io/pcd_io.h>
+#include <pcl/io/tim_grabber.h>
+
+#include <string>
+#include <random>
+
+using PointT = pcl::PointXYZ;
+using CloudT = pcl::PointCloud<PointT>;
+
+class TestableTimGrabber : public pcl::TimGrabber {
+  using TimGrabber::TimGrabber;
+  public: using TimGrabber::point_cloud_xyz_ptr_;
```

Comment:
Using one public is enough. (Hint: use struct which makes all the public redundant)

Reply:
Done